client/app/components/group/_pickupList/pickupList.controller.js

```diff
@@ -59,6 +57,15 @@ class PickupListController {
     this.filterAndDisplayPickups();
   }
 
+/**
+ * checks if a pickup is already full
```

Comment:
honestly this was auto completed by visual studio code .. but this is jsdoc i think. look here http://usejsdoc.org/tags-param.html